apps/api/v1/pages/api/event-types/[id]/_patch.ts

```diff
@@ -237,9 +237,9 @@ export async function patchHandler(req: NextApiRequest) {
 }
 
 async function checkPermissions(req: NextApiRequest, body: z.infer<typeof schemaEventTypeBaseBodyParams>) {
-  const { userId, isAdmin } = req;
+  const { userId, isSystemWideAdmin } = req;
   const { id } = schemaQueryIdParseInt.parse(req.query);
-  if (isAdmin) return;
+  if (isSystemWideAdmin) return;
```

Comment:
the logic inside `checkTeamEventEditPermission` only checks if it's a team event then user must be admin or owner of the team so org admin or owner not part of the team cannot edit event type.

Reply:
Event type logic hasn't been changed yet. It is the same as it was, and only considers system-wide admin.

REF:  ![image](https://github.com/calcom/cal.com/assets/52925846/e1ceec30-1b13-4e5e-87aa-a9a3823e6b4b)